modules/virt-configuring-secondary-network-vm-live-migration.adoc

```diff
@@ -0,0 +1,86 @@
+// Module included in the following assemblies:
+//
+// * virt/live_migration/virt-migrating-vm-on-secondary-network.adoc
+
+:_content-type: PROCEDURE
+[id="virt-configuring-secondary-network-vm-live-migration_{context}"]
+= Configuring a dedicated secondary network for virtual machine live migration
+
+To configure a dedicated secondary network for live migration, you must first create a bridge network attachment definition for a namespace in the CLI. Then add the name of the `NetworkAttachmentDefinition` object to the `HyperConverged` custom resource (CR).
```

Comment:
s/in the CLI/by using the CLI (I think "in the CLI" sounds like it's referring to the namespace, rather than the act of creation. This might be tricky localization-wise)

s/Then add/Then, add